helpers/controllers/submissionController.ts

```diff
@@ -103,25 +103,25 @@ export const rejectSubmission = async (
 export const submissions = async (
   _parent: void,
   arg: QuerySubmissionsArgs,
-  ctx: Context
+  _ctx: Context
 ) => {
-  try {
-    const { lessonId } = arg
-    await getReviewer(ctx.req.user, lessonId)
```

Comment:
Submissions can be viewed by reviewer (in this case they are coming from GetSubmissions query which checks if user has passed this lessons). And they can be viewed by student in this case they are coming from GetApp query which could be run by anyone. 

To update cache I need to refetch both of these. So I removed this check. It shouldn't cause problems, because review page has another check while accept/reject mutations still use getReviewer.

Reply:
Update. I found a way to make it work without removing `getReviewer` check. So student will refetch only GET_APP query while reviewer will refetch only GET_SUBMISSIONS query.